plugins/database/mongodb/mongodb_test.go

```diff
@@ -27,7 +26,7 @@ import (
 const mongoAdminRole = `{ "db": "admin", "roles": [ { "role": "readWrite" } ] }`
 
 func TestMongoDB_Initialize(t *testing.T) {
-	cleanup, connURL := mongodb.PrepareTestContainer(t, "5.0.10")
+	cleanup, connURL := mongodb.PrepareTestContainer(t, "latest")
```

Comment:
Do these tests still pass against v5 with the new code as well? Given the previous breakage, it might be worth running these tests against both v5 and latest.

Reply:
They also pass with _5.0.10_ tag